products/template/tests/test_build.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from dcpy.utils.postgres import get_schema_tables
+from .. import BUILD_SCHEMA
+
+
+# test build stage Load
+@pytest.mark.end_to_end()
+def test_load_db_tables():
+    expected_source_tables = [
```

Comment:
if you wanted to be fancy, you could read the recipes.yml and extract the expected datasets